internal/pkg/cli/app_show.go

```diff
@@ -257,7 +257,9 @@ func BuildAppShowCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "show",
-		Short: "Displays information about an application per environment.",
+		Short: "Shows settings of an application per environment.",
+		Long:  "Shows settings of an application, including task configuration, routes, and resources per environment.",
```

Comment:
What about something like this:

Shows info about a deployed app, including endpoints, capacity and related resources (per environment).